app/src/main/java/com/github/mik629/android/fundamentals/ui/movieslist/FragmentMoviesList.kt

```diff
@@ -40,6 +50,14 @@ class FragmentMoviesList : Fragment(R.layout.fragment_movies_list) {
     override fun onAttach(context: Context) {
         super.onAttach(context)
         appComponent.inject(this)
+        WorkManager.initialize(
```

Comment:
u can move all this code with enqueuing request exactly to UpdateCacheWorker companion